lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Controller     = require('./controller');
+const MovieValidator = require('../../../validators/movie');
+
+exports.register = (server, options, next) => {
+
+  server.route([{
+    method: 'POST',
+    path: '/movies',
+    config: {
+      handler: function (request, reply) {
+        reply(Controller.create(request.payload));
+      },
+      validate: {
+        payload: MovieValidator
+      }
+    }
+
+  }]);
+
+  next();
+
```

Comment:
We're having them [pad the `next` on both sides with a blank line](https://github.com/lob/miyagi/blob/master/06-build-your-own-api/10-endpoint-configuration.md#4-define-the-post-movies-route), at least in miyagi, but I'm not sure if we've really discussed our style preference.